demos/JWST/S3_nirspec_fs_template.ecf

```diff
@@ -25,6 +25,9 @@ prof_deg        3           # Polynomial degree, when fittype = poly
 p5thresh        10          # X-sigma threshold for outlier rejection while constructing spatial profile
 p7thresh        60          # X-sigma threshold for outlier rejection during optimal spectral extraction
 
+# G395H curvature treatment
+curvature       correct     # How to manage the curved trace on the detector (Options: None, correct)
```

Comment:
Can you also update this in the testing ECF and the ECF in docs/media. Also, please document this new setting in docs/source/ecf.rst

Reply:
Yes, should be updated everywhere now!